docs/_docs/testnet/starting-a-node.md

```diff
@@ -0,0 +1,51 @@
+---
+---
+# Starting a node
+How to start a node in the Evan's Cat testnet.
+
+## Requirements
+- SGX enabled VM
+- Docker
+
+## Steps
+#### - Create an SGX enabled VM
+Recommended Standard DC4s v2 (4 vcpus, 16 GiB memory) in Azure.
+
+#### - Install Docker
+
+```
+sudo apt-get update \
+    && sudo apt-get install -y jq \
+    && curl -fsSL https://get.docker.com -o get-docker.sh && sh ./get-docker.sh
+```
+
+#### - Download Obscuro Start script
+
+Make sure to use the latest version at https://github.com/obscuronet/go-obscuro/tags
+
+```
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/start-obscuro-node.sh && \
+  wget https://raw.githubusercontent.com/obscuronet/go-obscuro/v0.7.1/testnet/docker-compose.yml
+```
+
+#### - Start Obscuro Node
+
+To start the obscuro node some information is required to populate the starting script.
+
+- (p2p_public_address) Local address for the Peer to Peer connections. Must be open to outside connections.
```

Comment:
Where does a user get this information from?

Reply:
How does this sound ?

```suggestion
- (p2p_public_address) The external facing address of the network. Where outside peers will connect to. Must be open to outside connections.
```

Reply:
But what is that i.e. where do I find the actual value for testnet or dev-testnet. In your example on the epic you had --p2p_public_address="51.142.241.27:10000" ... did you go into azure to get that?